charts/topolvm/Chart.yaml

```
@@ -12,7 +12,7 @@ sources:
 dependencies:
   - name: cert-manager
     repository: https://charts.jetstack.io
-    version: 1.7.0
+    version: 1.16.1
```

We have decided to use the LTS version of cert-manager.
ref. https://cert-manager.io/docs/releases/#currently-supported-releases

We have also decided to follow [the best practice of the helm chart](https://helm.sh/docs/chart_best_practices/dependencies/#versions) and use the version range instead of the exact version.

Could you change the code as follows?
```suggestion
    version: ~1.12.x
```

cf. https://github.com/Masterminds/semver?tab=readme-ov-file#tilde-range-comparisons-patch

---

@peng225 looks good to me, I've updated the commit. What bothers me a little about version 12 is the known error (https://cert-manager.io/docs/releases/release-notes/release-notes-1.12/#known-issues) but since we are working with multiple secrets here, that should be fine.

---

Could you run `helm dependency update charts/topolvm/` to upgrade `charts/topolvm/Chart.lock`? Also, it would be great if you could squash your commits into one. (If you're not sure how to do that, you can check out https://stackoverflow.com/a/5201642, for example.) Thanks!

---

@ushitora-anqou OK, I've updated the Chart.lock, too and squashed the commits into one, thank you for that hint!